### PR TITLE
add graphicdesign.SE room for alerts

### PR DESCRIPTION
**Pull request by tripleee**
http://chat.stackexchange.com/rooms/56223/the-spam-blot as discussed here:
http://chat.stackexchange.com/transcript/message/36356365#36356365

### CODE REVIEW

**Comment by tripleee** [comment]
Updated https://github.com/Charcoal-SE/SmokeDetector/wiki/Chat-Rooms

**Comment by angussidney** [comment]
Awating definitive confirmation: http://chat.stackexchange.com/transcript/message/36356765#36356765

File: ws.py [hunk 1]
```diff
@@ -229,6 +229,11 @@
         "sites": ["bricks.stackexchange.com"],
         "room": GlobalVars.wrap.get_room("1964"),
         "unwantedReasons": []
+    },
+    {
+        "sites": ["graphicdesign.stackexchange.com"],
+        "room": GlobalVars.wrap.get_room("56223"),
```
Comment (angussidney) [inline ws.py]: Still awaiting final confirmation, they are considering a meta post first.

**Comment by Cerbrus** [comment]
I'll keep an eye on this. If he's added to the GD room, I'll update FIRE / AIM to work in there as well.  
If I somehow overlook this, give me a slap on the wrist :-)

**Comment by tripleee** [comment]
The GD admins still want to defer.  http://chat.stackexchange.com/transcript/message/36464416#36464416

**Comment by teward** [comment]
If the GD admins have chosen to defer, then we will have to wait until they choose to actually go ahead and "jump" on this.  If they choose to actually go ahead with it, then just add them back into the alerts list with a *new* pull request, please.

**Pull request merged**